app/lib/multistep/controller.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+module Multistep
+  module Controller
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def form(value = nil, as: nil) # rubocop:disable Naming/MethodParameterName
+        if value
+          @form = value
+          if as
+            alias_method as, :form
+            helper_method as
+          end
+        else
+          @form
+        end
+      end
+
+      def session_key(value = nil)
+        if value
+          @session_key = value
+        else
+          @session_key ||= @form && @form.model_name.param_key
+        end
+      end
+
+      def params_key(value = nil)
+        if value
+          @params_key = value
+        else
+          @params_key ||= @form && @form.model_name.param_key
+        end
+      end
+
+      def escape_route(&block)
```

Comment:
Can we call it `default_back_route` or something like that? `escape` can mean many many things.

Reply:
Yeah, I wasn't happy with the name, `default_back_route` sounds much better.